### PR TITLE
Settting backend connection

### PR DESCRIPTION
**Pull request by dalshekerchi**
Connected setting activity to backend by using the database and changing the quaries as needed. Added the ability for the user to edit information that they signed up with.

### CODE REVIEW

**Review by sarihammad — COMMENTED** [review]
Looks good!

**Review by sarihammad — DISMISSED** [review]
Looks good!

File: Uni_life_tracker/app/src/main/java/com/generic/ult/SettingsActivity.java [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
 
 public class SettingsActivity extends AppCompatActivity {
     // initialize variable
     DrawerLayout drawerLayout;
 
+    private TextInputEditText etName;
+    private TextInputEditText etUni;
+    private TextInputLayout tiName;
+    private TextInputLayout tiUni;
+
```
Comment (hellokathylee) [inline Uni_life_tracker/app/src/main/java/com/generic/ult/SettingsActivity.java:21]: a really minor detail, but if you want you can declare these all in 1 line to save a few lines

**Review by hellokathylee — DISMISSED** [review]

**Review by sarihammad — DISMISSED** [review]
Looks good!

**Review by hellokathylee — DISMISSED** [review]

**Review by hellokathylee — APPROVED** [review]
looks great!!

**Review by sarihammad — APPROVED** [review]
looks great!

**Pull request merged**